libafl/src/feedbacks/mod.rs

```diff
@@ -929,6 +929,60 @@ impl Default for TimeoutFeedback {
 /// A feedback factory for timeout feedbacks
 pub type TimeoutFeedbackFactory = DefaultFeedbackFactory<TimeoutFeedback>;
 
+/// A [`DiffExitKindFeedback`] checks if there is a difference in the [`crate::executors::ExitKind`]s in a [`crate::executors::DiffExecutor`].
+#[derive(Serialize, Deserialize, Clone, Debug)]
+pub struct DiffExitKindFeedback {}
```

Comment:
I think you could do `pub struct DiffExitKindFeedback;`

Reply:
I did it the same way as in `TimeoutFeedback`. But agreed, I'm not sure why the extra code is necessary. Want me to change it for the other `Feedback`s in this file as well?

Reply:
Yes please :)